go.mod

```diff
@@ -162,6 +164,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
+	golang.zx2c4.com/wireguard v0.0.0-20231211153847-12269c276173
```

Comment:
probably needs to go to main require block

Reply:
Done!